app/views/categories/index.html.erb

```diff
@@ -0,0 +1,55 @@
+<section class='categories-page f-height flex'>
+  <header class='flex f-width'>
+    <nav class='flex row f-width'>
+      <h1>Categories</h1>
+      <%= button_to destroy_user_session_path, method: :delete do %>
+        <span class="material-symbols-outlined">
+          logout
+        </span>
+      <% end %>
+    </nav>
+  </header>
+
+  <div class='total-amount flex row f-width'>
+    <p>Total amount</p>
+    <p>$ <%= @total.reduce(:+) %></p>
+  </div>
+
+  
+  <div class='categories-container f-width'>
+    <% if @msg_bool %>
+      <p class='temp-msg'>
+        <em>Please add a category in order to add a transaction</em>
+      </p>
+    <% end %>
+    <% @categories.each do |item| %>
+      <%= link_to category_dealings_path(item.id) do %>
+        <div class='item-card flex row f-width'>
+          <img
+          class='item-img'
+          src=<%= item.icon %> 
+          alt='item icon'
+          >
+          </img>
+          <h3 class='item-header'><%= item.name %></h3>
+          <p class='item-creation-date'>
+            $ 
+            <% if !item.total %>
+              <%= 0 %>
+            <% else %>
+              <%= item.total %>
+            <% end %>
+          </p>
+        </div>
+      <% end %>
+    <% end %>
+  </div>
+
+  <div class='create-container'>
+    <%= link_to new_category_path do %>
+      <button type='button' name='create_category' class='btn f-width'>
```

Comment:
- [ ] Again, this button looks awesome. Unfortunately, you should follow the design guidelines as mentioned before.
### Your Implementation
![image](https://user-images.githubusercontent.com/45460192/227736072-015a66cb-1857-47b7-81c1-d58b64140f40.png)
### Original
![image](https://user-images.githubusercontent.com/45460192/227736099-d3114ee7-88f0-4929-aa05-ab67a5f934bd.png)